flw/src/main/resources/META-INF/spring/blueprint.xml

```diff
@@ -33,6 +33,9 @@
     <osgi:service ref="whitelistService"
                   interface="org.motechproject.nms.flw.service.WhitelistService" />
 
+    <osgi:service ref="itService"
```

Comment:
Or something like having the it service disable itself from a settings?
So we don't have to recompile